recipes/py4xs/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "py4xs" %}
+{% set version = "2021.6.2.1" %}
+{% set sha256 = "052eda0c273e38844f1e828e9927abfdec4fe80041c81a8acff75e57b10b3db7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - fabio
+    - h5py
+    - matplotlib
+    - numpy
+    - pillow
+    - scipy
+
+test:
+  imports:
+    - py4xs
```

Comment:
Same here, `__init__` does nothing so importing some other modules would be a much better test that things are right here.